docs/privacy.md

```diff
@@ -0,0 +1,188 @@
+- [What are your data protection rights?](#what-are-your-data-protection-rights)
+- [What log data do we collect?](#what-log-data-do-we-collect)
+- [Do we use cookies?](#do-we-use-cookies)
+- [Do we use any third party cookies?](#do-we-use-any-third-party-cookies)
+- [Do we share your personal data with third parties?](#do-we-share-your-personal-data-with-third-parties)
+- [Privacy policies of other websites](#privacy-policies-of-other-websites)
+- [Changes to our privacy policy](#changes-to-our-privacy-policy)
+- [How to contact us](#how-to-contact-us)
+
+
+## Introduction
+
+Thank you for choosing to be part of our community at UserStyles.world (“we”,
+“us”, or “our”).
+
+When you visit our website userstyles.world (“Site”) and use
+our services, you trust us with your personal information. We take your privacy
+very seriously.
+
+We are committed to protecting your personal information and your right to
+privacy. If you have any questions or concerns about our policy, or our
+practices with regards to your personal information, please contact us at
+[feedback@userstyles.world](mailto:feedback@userstyles.world).
+
+In this privacy notice, we describe our privacy policy. We seek to explain to
+you in the clearest way possible what information we store, how we use it and
+what rights you have in relation to it. We hope you take some time to read
+through it carefully, as it is important.
+
+If there are any terms in this privacy policy that you do not agree with, please
+discontinue the use of our Site and our services.
+
+This privacy policy applies to all information stored through our Site, and/or
+any related services. Please read it carefully as it will help you make informed
+decisions about sharing your personal information with us.
+
+
+## What data do we collect?
+
+We collect the following data:
+
+- Username
+- Email address
+- Links to other platforms
+- Visited "userstyle details" pages
+- Visited "userstyle install" pages
+
+Additionally, we also collect the data supplied by OAuth providers. However, we
+only use the email address and username in order to create your account and
+discard the rest of the data that was provided.
+
+
+## How do we collect your data?
+
+All of the data is directly provided by your activity on the Site. The data is
+processed when you:
+
+- Register a new account
+- Visit userstyle details page (e.g. [https://userstyles.world/style/1/userstyles-world-tweaks][details])
+- Visit userstyle install page (e.g. [https://userstyles.world/api/style/1.user.css][api])
+
+[details]: https://userstyles.world/style/1/userstyles-world-tweaks
+[api]: https://userstyles.world/api/style/1.user.css
+
+
+## How do we use your data?
+
+We use your data in order to show userstyle statistics.
+
+
+## How do we store your data?
+
+We follow best practices when it comes to handling your data and store
+everything on our own server.
+
+The data used for userstyle statistics is anonymizing the IP address through a
```

Comment:
Read this sentence up until hash function. It reads smooth rif you have the hash function... Anomymzed IP Adress etc